### PR TITLE
default build with MKL for desktop

### PR DESCRIPTION
**Pull request by bes-dev**
default build with MKL for desktop

### CODE REVIEW

**Comment by ezyang** [comment]
Hi @BelBES, can you tell us a little more about what this patch does?

**Comment by BelBES** [comment]
@ezyang Sorry for noninformatiable PR. So, currently, if Intel MKL installed in system it's not be used for build with pytorch because 'emt64t' architecture was been renamed to 'intel64'.

File: aten/src/TH/cmake/FindMKL.cmake [hunk 1]
```diff
@@ -63,7 +63,7 @@ ENDIF (CMAKE_COMPILER_IS_GNUCC)
 # Kernel libraries dynamically loaded
 SET(mklkerlibs "mc" "mc3" "nc" "p4n" "p4m" "p4m3" "p4p" "def")
 SET(mklseq)
-
+SET(INTEL_COMPILER_DIR "/opt/intel")
```
Comment (soumith) [inline aten/src/TH/cmake/FindMKL.cmake]: you dont need to add this line. you should add this dir to `CMAKE_PREFIX_PATH` environment variable

**Comment by soumith** [comment]
the change from emt64t to intel64 looks good, but the second line change is not necessary. I made inline comment.

**Review by soumith — APPROVED** [review]

**Comment by soumith** [comment]
thanks @BelBES !

**Pull request merged**